usaspending_api/references/management/commands/load_naics.py

```diff
@@ -32,26 +32,45 @@ def handle(self, *args, **options):
 
 def populate_naics_fields(ws, naics_year, path):
     for current_row, row in enumerate(ws.rows):
-        if current_row == 0:
-            continue
-
         if not row[0].value:
             break  # Reads file only until a blank line
 
+        naics_desc = row[1].value.strip()
+
         try:
             naics_code = int(row[0].value)
+            load_single_naics(naics_code, naics_year, naics_desc)
+        # Occasionally you will see more "creative" ways of listing naics. The following tries to account for common
+        # patterns
         except ValueError:
-            raise CommandError("Invalid NAICS code: {0}. Please review file {1}".format(naics_code, path))
+            load_naics_range(row[0].value, naics_year, naics_desc, path)
 
-        naics_desc = row[1].value.strip()
 
-        obj, created = NAICS.objects.get_or_create(
-            pk=naics_code, defaults={"description": naics_desc, "year": naics_year}
-        )
+def load_naics_range(naics_range_string, naics_year, naics_desc, path):
+    if "-" in naics_range_string:
+        try:
+            minmax = naics_range_string.split("-")
+            for naics_code in range(int(minmax[0].strip()), int(minmax[1].strip()) + 1):
+                load_single_naics(naics_code, naics_year, naics_desc)
+        except ValueError:
+            raise CommandError(
+                "Unparsable NAICS range value: {0}. Please review file {1}".format(naics_range_string, path)
+            )
+    else:
+        raise CommandError("Unparsable NAICS range value: {0}. Please review file {1}".format(naics_range_string, path))
+
+
+def load_single_naics(naics_code, naics_year, naics_desc):
+
+    # crude way of ignoring naics of length 3 and 5
+    if len(str(naics_code)) not in {2, 4, 6}:
```

Comment:
This is ok and will work, but traditionally we would use a tuple here `(2, 4, 6)`.   Tuples are lighter weight than sets.